### DISCUSSION TITLE
Dealing with Anesthesia Services

### DISCUSSION THREAD

**Discussion by ryan-thomas-peterson** [CMSgov/price-transparency-guide#62] [Q&A] [answered]
How should time based services be reported, such as Anesthesia services?

**Comment by shaselton-usds** [comment]
I would imagine each time-based services is broken down to a specific code, correct? You would be reporting on those codes that represent discrete time frames.

**Reply by ryan-thomas-peterson** [reply]
We could look at, for example 00811 for an Anesthesia code for an endoscopic procedure.  Each anesthesia code has an underlying number of CMS 'base' units (which are 15 minute intervals).  For this procedure there are 4 base units.  

However the services can (and are for us)  paid per 'minute' used which will vary provider to provider and service to service.  

Absent guidance I guess we'd take a median time from observed history per provider and take an overall median for the providers that we suspect could perform the service but that we haven't seen yet.

**Comment by ghost** [comment]
Yes, that is one approach we are looking at
- basing the units off historical claim data from our enterprise data warehouse for the billing code, or
- a concept we call "rule of 1" being 1 base unit would be used to calculate the negotiated rate for any billing codes involving units absent any guidance

**Reply by tdfow** [reply]
I would think this is similar to the percentage of charge scenario.  Wouldn't the schema need to support a notation or something to indicate that the negotiated rate's basis info?  For example, a notation or basis option to indicate, unit, day, charge amount, etc?    Otherwise, how would the negotiated rate be interpreted or useable?   The mandate indicates references to 'notations' being in the file---so, would there be an expectation to include the basis info of the rate?  If rates are being published based on x, then providing basis info identifies how the rate would vary.

**Reply by ghost** [reply]
@tdfow I agree. For the example of charge amounts, we plan on pulling historical data for the average charge amount for billing codes that require as a factor of the calculation of negotiated rate since we could not use the rule of 1 for $1 as the charge amount. But then again basing the historical charge amount opens up the questions of how that value is obtained vs someone else (back to the same theme we have commented in our questions/discussions). 

The other headache is where there is no historical claim reference for a charge amount say for a obscure code never provided in your area or the case when a new billing code is provided since we are required to provide a negotiated rate for all items/services for the In-Network Negotiated Rates file.

Use of the 'notations' to call out some of this information would be helpful to allow a better understanding of how a rate is calculated. Or not limiting the negotiated rate to being only a dollar amount but something more flexible?

**Reply by nmayle** [reply]
Do similar "unit" and "quantity" issues come with drugs in the in-network charges file?  Also, NDC and HCPCS can have different units, so without the units it can be ambiguous from the codes alone as to what amount of the drug corresponds to the listed price.

**Reply by ghost** [reply]
@nmayle Definitely for drugs because you are then asking for a negotiated rate then at what unit level (UN, ML, GR, F2) value to be reported? Do you assume then a unit of 1 GR or 10 GR of a substance? Goes back to how @tdfow calls out the reference to 'notations'.

**Reply by ryan-thomas-peterson** [reply]
@shaselton-usds , are these concepts of notations being considered to address these use cases?

**Reply by reevesj2** [reply]
For Anesthesia we also pay units for time (15 minutes=1 unit).  My thought was to use the ASA Base + 1 unit.  If possible to notate then the notation would indicate Anesthesia is based on ASA units +1 Time Unit (15 minutes)  X Negotiated Unit Rate.  That way someone would be able to come up with a good estimate of the cost based on their time units if they needed to.    

Percent of charge I was thinking of doing a similar calculation where the assumed charge for every item is $100, the notation would be Pricing is based on a sample charge of $100 * negotiated percent age.  So a service that would be pay 80% would end up looking like 80.00 in the file. Hopefully someone that is decent in math will be able to figure out the percentage to use for calculating their cost.

**Comment by jltang-kp** [comment]
Bumping this post up again, to echo @ryan-thomas-peterson. We would like some direction in cases where we don't know unit of time until the services are performed. 

Is the guidance to put a representative amount for units? 

@BobSyracuse suggestion of rule of 1 it sounds like some sort of disclaimer language would be needed as well

**Reply by ryan-thomas-peterson** [reply]
Given the lack of guidance on these we've opted to produce rates based upon the base units associated with the code and include that in the code description.

**Comment by ryan-thomas-peterson** [comment, marked as answer]
With support of Additional Information fields we will be using this but acknowledge this catch all will make it harder to compare things between file producers for consumers of the file.  Flagging as answered.
